### ISSUE TITLE
Creation of sd.ini on Mac

### ISSUE THREAD

**Issue by nigelgentry** [mpogue2/SquareDesk#386]
I mentioned this issue to Dan when we met in Seattle.

The SD module tells me that I need to initialise a session file.  But when I try the sd command "initialize session file", I get an error saying that the sd.ini file cannot be created.

<img width="401" alt="image" src="https://user-images.githubusercontent.com/41304916/42776821-16aaa1d8-88ed-11e8-8d3b-9d086631fdea.png">

I have made the SquareDeskMusic folder read and writable by everyone.  I get the same result whether running the application from the Applications folder or the SquareDeskMusic folder.

Do I actually need this file to run the SD module?

I am using a MacBook Pro Retina (Late 2012) running MacOS 10.13.5.

**Comment by mpogue2** [comment]
Ah...that is an artifact of SD that we accidentally expose through the "SD Output" tab.  I view that more like "here's a bunch of under-the-covers information that SD experts might want to see".  In general, the other tab, the "Current Sequence" tab is what most people will use.

I suspect that on Mac it is trying to write into the directory that is INSIDE the application bundle (inside the squaredesk.app file), and that probably fails.

I don't think you need that file at all to run the SD module.  I never use it, and I use SD all the time.
In general, you should probably avoid using any SD command that tries to write files somewhere.... :-)

SquareDesk is somewhat of a "wrapper" around SD, rather than taking over SD entirely and bending it to our will.  This makes it much easier for us to take a new version into SquareDesk, but it also means that some wackiness happens when certain SD commands are used....

TL;DR: OK to ignore it.

**Comment by danlyke** [comment]
I think, though, that I need to figure out where SD is trying to write this and make sure I figure a way to write it somewhere else. It's gotta be possible... and I do want to encapsulate SD as much as I can.

Though I also want to get to the point where we're saving and organizing SD sequences internally.

**Comment by mpogue2** [comment]
I dunno...I suspect there are other things that SD writes that could also mess things up... :-). On Mac, I am suspecting that it's trying to write into the App Bundle, since that's where SD is located, but I don't know for sure...I would say that manually typing "initialize session file" at least for now is not supported.

**Comment by mpogue2** [comment]
Same as #379, I think.

**Comment by mpogue2** [comment]
"initialize session file" is still not supported, on any platform.  So, I'm closing this one for now.
It should not cause a problem if somebody does it, AFAIK, I'm just trying to avoid writing into the Contents/MacOS folder, if at all possible.

**Comment by mpogue2** [comment]
Closing for now.
